### PR TITLE
feature: restaurant-page

### PR DESCRIPTION
**Pull request by Mindo-Joseph**
### What does this PR do?
Creates a restaurant page using JavaScript by making use of different modules through webpack

### Description of Task to be completed?
The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack.

### How should this be manually tested?
- Install dependencies by running ```npm i``` in your terminal
- Run webpack build by using the following command ```npm run build``` 
- Once you have installed dependencies and run webpack build command  open ```dist/index.html``` in your browser
- Click on the nav bar links to test that the view is tabbed

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### HI Mindo 👋

**Nice work so far on this. 👌**

However, let's make the following changes:

- [ ] Please include the project live link in your readme along with the screenshot. Consider using Github pages follow [this link](https://gist.github.com/cobyism/4730490) for more info.

- [x] Rather than regular functions please use arrow functions. Change this on all affected modules.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "restaurant",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (mikenath223) [inline package.json]: - [x] Kindly remove test, build and watch commands from script and replace them with the following commands "dev", "build" and "start".

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  restaurantName.textContent = 'Eatery Menu';
+
+  const menuItems = document.createElement('div');
+  menuItems.className = 'row';
+  const leftSide = document.createElement('div');
+  leftSide.className = 'column';
+
+  const foods = [['Expresso', 'Americano', 'Machiato', 'Cappuccino'], ['French Fries', 'Italian Pizza', 'Spaghetti', 'Burger'], ['Olong', 'English BreakFast', 'Jasmine Green', 'Emerald Spring', 'King Crimson', 'Blue Crane'], ['Soft Drink', 'Banana Berry', 'Tropical Passion', 'Hawaian Breeze', 'Sunset', 'Apple Crush']];
+  // eslint-disable-next-line max-len
+  const prices = [[3.00, 4.00, 3.50, 5.00], [5.00, 17.00, 12.00, 8.00], [3.00, 4.00, 3.00, 3.00, 5.00, 5.50], [3.00, 2.00, 4.00, 2.00, 3.00, 3.00]];
```
Comment (mikenath223) [inline src/menu.js]: - [x] Please break these lines at commas(,) as this won't affect the code running, and then you can remove the esint-disable rule.

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon:  

Hi, @Mindo-Joseph, congratulations for fixing all the comments outlined by the previous TSE :+1: Great job :slightly_smiling_face: Go ahead and merge into development

Good luck with your next projects :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**